### PR TITLE
tarsnap, tarsnap-keygen: add pages

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.0.40

For #11901.

### CODE REVIEW

File: pages/common/tarsnap.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# tarsnap
+
+> Manipulate remote Tarsnap encrypted backups.
+> NOTE: you don't need to specify the key file and the cache directory if you configure them on `/usr/local/etc/tarsnap.conf` and/or `~/.tarsnaprc`.
```
Comment (kbdharun) [inline pages/common/tarsnap.md:4]:
```suggestion
> Note: you don't need to specify the key file and the cache directory if you configure them on `/usr/local/etc/tarsnap.conf` and/or `~/.tarsnaprc`.
```

File: pages/common/tarsnap.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+> See also: `tarsnap-keygen`.
+> More information: <https://www.tarsnap.com/man-tarsnap.1.html>.
+
+- [c]reate a backup archive of one or more files or direcotries, specifying the cryptographic key and the cache directory:
```
Comment (kbdharun) [inline pages/common/tarsnap.md:8]:
```suggestion
- [c]reate a backup archive of one or more files or directories, specifying the cryptographic key and the cache directory:
```

**Review by kbdharun — APPROVED** [review]

File: pages/common/tarsnap.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# tarsnap
+
+> Manipulate remote Tarsnap encrypted backups.
+> Note: you don't need to specify the key file and the cache directory if you configure them on `/usr/local/etc/tarsnap.conf` and/or `~/.tarsnaprc`.
```
Comment (acuteenvy) [inline pages/common/tarsnap.md:4]: Only one config file is read.

https://www.tarsnap.com/man-tarsnap.1.html
>/usr/local/etc/tarsnap.conf
             The system global tarsnap configuration file.  ***Parameters
             specified here only take effect if they are not specified via the
             current user's local configuration file*** or via the command line.
```suggestion
> Note: you don't need to specify the key file and the cache directory if you configure them in `/usr/local/etc/tarsnap.conf` or `~/.tarsnaprc`.
```

File: pages/common/tarsnap.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+- [d]elete a specific archive:
+
+`tarsnap -d --keyfile /usr/tarsnap.key --cachedir /usr/tarsnap-cache -f backup-2008-04-24`
+
+- Lis[t] the contents of a specific archive in [v]erbose mode:
+
+`tarsnap -tv --keyfile {{path/to/key_file}} -f {{archive_name}}`
+
+- Restore specific one or more files or directories from an specific archive:
```
Comment (acuteenvy) [inline pages/common/tarsnap.md:28]:
```suggestion
- Restore one or more files or directories from a specific archive:
```

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**